.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Do not commit IDE specific files to Git. Delete it and add .idea folder to .gitignore.